doc/github-org-owners.md

```diff
@@ -0,0 +1,49 @@
+## GitHub org owners
+
+The following people have the GitHub "owners" permissions on the NixOS organization:
+<!-- Also keep this in sync with the members of @NixOS/org! -->
+- [@infinisil](https://github.com/infinisil)
+- [@lassulus](https://github.com/lassulus)
+- [@tomberek](https://github.com/tomberek)
+- [@winterqt](https://github.com/winterqt)
+- [@zimbatm](https://github.com/zimbatm)
+## How to contact the team
+For any GitHub-related needs, you can reach out to the org owners by either:
+- Pinging [@NixOS/org](https://github.com/orgs/NixOS/teams/org)
+- [Creating an issue in this repository](https://github.com/NixOS/org/issues/new).
+- Messaging in the [Github org owners help desk Matrix room](https://matrix.to/#/#org_owners:nixos.org).
+
+### Authority and processes
+This team's role is to manage and unblock users of the github.com/NixOS GitHub organization. The @NixOS/steering gives them autonomy to handle small day-to-day tasks and expects them to escalate bigger decisions.
+All org owners can individually take care of implementing:
+- Decisions by bodies that have the authority to make GitHub org changes such as:
+  - Arbitrary decisions by the Steering Committee.
+  - Moderation decisions by the moderation team.
+  - Changes from approved RFCs.
+- Low-impact changes, such as:
+  - Adding new org members to allow review requests.
```

Comment:
How do you imagine this to be used? In Nixpkgs, this is handled by [commiter delegation team](https://github.com/NixOS/org/blob/main/doc/github.md#nixpkgs), and in other repos like Nix, I would expect them to have owners who would ideally self-organize.

Reply:
This is just for the issue of people missing the automated org invite they get after adding themselves to `lib.maintainers`, see https://github.com/NixOS/nixpkgs/issues/234293. An org owner currently needs to resend these missed invites, since there's no automation for that. These invites give no commit access, they're just to allow requesting reviews from maintainers.